packages/components/ng-ovh-mondial-relay/src/index.js

```diff
@@ -0,0 +1,23 @@
+import 'leaflet';
+
+import angular from 'angular';
+import 'angular-translate';
+import 'angular-leaflet-directive';
+import 'ovh-api-services';
+
+import component from './component';
+
+import './index.less';
+
+const moduleName = 'ngOvhMondialRelay';
+
+angular
+  .module(moduleName, [
+    'leaflet-directive',
+    'ovh-api-services',
+    'pascalprecht.translate',
+  ])
+  .component('ovhMondialRelay', component)
+  .run(/* @ngTranslationsInject:json ./translations */);
```

Comment:
Since translations files has moved from XML to JSON, I've added the format to the
following annotation `@ngTranslationsInject:json`.